packages/agent/src/display.ts

```diff
@@ -484,6 +491,9 @@ export function getCredentialForDisplay(credentialRecord: W3cCredentialRecord |
       validFrom: undefined,
     } satisfies CredentialMetadata,
     claimFormat: credentialRecord.credential.claimFormat,
+    validUntil: credentialRecord.credential.expirationDate
```

Comment:
No validFrom? I think we can use issuanceDate, as they renamed that to validFrom in v2 of w3c credential spec.